README.md

```diff
@@ -14,6 +14,23 @@ By default there is little/no output to stdout or stderr until the script has fi
 
 When using `jellyfin.py`, the results can be saved to `json` using the `-j` parameter. These will be saved in a sub-directory in `pwd`. Saving the results as json also allows them to be checked in subsequent runs to skip already processed files.
 
+## Running Docker
+  ### Scanner
+  ```
```

Comment:
This looks good!

At some point I'd like to have a docker compose example too, so if you want to add that it would be cool, but a docker run is also fine for now